### DISCUSSION TITLE
Low-end tier pricing in the future?

### DISCUSSION THREAD

**Discussion by mattwalshdev** [planetscale/discussion#445] [Feature Requests] [answered]
The hobby scale free tier is great, but would prefer to have a few small db's under the same account and support the company in a small way.
The Scalar db offering is a bit too pricey for the size of my projects, everything I run can be done on the hobby level, any chance of offering additional paid db's with the hobby level I/O limits? Something like $5-$10 per db a month?

**Comment by fooforge** [comment, marked as answer]
Hey @mattwalshdev,

This is good feedback and something we hear regularly. Thank you!

Especially for agencies working on lots of projects where each project needs their own small isolated database, we're not the best fit yet. It is something we have on the radar and that we would like to improve, but I can't share any specifics publicly yet, I'm afraid.
